### ISSUE TITLE
-Mektup Sistemi-

### ISSUE THREAD

**Issue by erenbulbul** [hsndmrts/RPG#10]
Mektup göndermek istediğiniz bir kişi varsa bunun sayesinde mektup gönderebilirsiniz.
Kullanım ücreti 2 Altındır. Mektup hedefe 1-2 dakika sonra gider.

**Kullanım:**
/mektup <Hedef> <Mesaj>

**Örn:**
/mektup Yamalidon Valim, Şehirde büyük bir kargaşa çıktı. Yardımınız gerek.

**Vali'nin 15 blok etrafında gözüken.**
[Bilgi] Bir atlı, Yamalidon'a mektup getirir.

**Vali'nin gördüğü:**
[Mektup] Ahmet: Valim, Şehirde büyük bir kargaşa çıktı. Yardımınız gerek.

**Vali mektubu tekrar okumak isterse** (Gerekli değil. Yapabiliyorsan yap.)
/SonMektup

**Comment by hsndmrts** [comment]
oyuncuya ne kadar uzaksa o kadar zor gitsinmi? yoksa direk 1-2 dk mı

**Comment by erenbulbul** [comment]
Evet. Yakındaysa 1 dk, 1000 blok mesafedeyse 2 dk, 2000 blok mesafedeyse 3 dk.

**Comment by erenbulbul** [comment]
Her 1 blok uzaklık için 0.06 saniye geç gitsin.

**Comment by hsndmrts** [comment]
anladım yani milisaniye il 6tick oluyor her block ta
